modules/xfem/test/tests/moving_interface/verification/tests

```diff
@@ -0,0 +1,90 @@
+[Tests]
+  [./1D_xy_homog1mat]
+    type = Exodiff
+    input = 1D_xy_homog1mat.i
+    exodiff = '1D_xy_homog1mat_out.e 1D_xy_homog1mat_out.e-s002 1D_xy_homog1mat_out.e-s003 1D_xy_homog1mat_out.e-s004 1D_xy_homog1mat_out.e-s005'
+    map = false
+    unique_id = true
+    allow_test_objects = True
+    requirement = 'The LevelSetCutUserObject class shall be able to be used to verify that the XFEM module is capable of accurately solving 1D, xy problems with homogeneous material properties with a moving interface developed via the Method of Manufactured Solutions.'
```

Comment:
I think I'd drop the first part of the sentence and just say something like:

The XFEM module shall accurately solve 1D, xy problems with homogeneous material properties with a moving interface as demonstrated using the Method of Manufactured Solutions.

Reply:
Updated requirements for all verification tests.